src/olympia/api/tests/test_permissions.py

```diff
@@ -169,6 +170,19 @@ def test_has_object_permission_anonymous(self):
         )
 
 
+class TestAllowAddonAuthorIfNotMozillaDisabled(TestAllowAddonAuthor):
+    def setUp(self):
```

Comment:
Is it worth having the `setUp` if there's only one test case? Or is it better for have it for consistency?

Reply:
The class doesn't have only one test - it inherits from `TestAllowAddonAuthor`